deployments/ansible/roles/collector/tasks/otel_win_install.yml

```diff
@@ -61,12 +61,91 @@
   set_fact:
     msi_arguments: "{{ msi_unfiltered_arguments | reject('eq', '') | list }}"
 
+- set_fact:
```

Comment:
In testing, if variables without spaces have double quotes, MSI installation fails. This logic then separates arguments into two lists, one that has spaces, one without. The args without spaces are left unchanged. Arguments with spaces have the value surrounded by double quotes. After this, the lists are then combined again into the `msi_arguments` fact.